packages/web-components/src/components/ic-breadcrumb/ic-breadcrumb.tsx

```diff
@@ -17,20 +17,20 @@ export class Breadcrumb {
 
   @Element() el: HTMLIcBreadcrumbElement;
   /**
-   * Set breadcrumb title
+   * The breadcrumb title.
```

Comment:
Just my opinion really but maybe this instead? 
```suggestion
   * The title of the breadcrumb.
```
I think the wording would sound more consistent with other descriptions